doc/lsst.ctrl.bps/quickstart.rst

```diff
@@ -1244,7 +1243,7 @@ The major differences to users are:
     the output run in the provided pre-existing quantum graph.
   - ``final_post_finalJob.out``: An internal file for debugging incorrect
     reporting of final run status.
-  - ``<qgraph_filename>_orig.qgraph``: A backup copy of the original
+  - ``<qgraph_filename>_orig.qg``: A backup copy of the original
```

Comment:
Just a note that if the original file is a `.qgraph` the copy will also be a `.qgraph` instead of a `.qg`.   Assuming we're really not striving for backward compatibility for very long, I don't know that anyone will really notice.